examples/create-react-app/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "16.8.4",
+    "react-dom": "16.8.4",
+    "react-scripts": "2.1.8"
```

Comment:
should `graphql-hooks` and `graphql-hooks-memcache` be in here?

Reply:
Missed these, thanks!